.bento/archive.json

```diff
@@ -1,6 +1,14 @@
 {
   "bandit": {
     "violations": {
+      "997ed2dbfab1051ab8e97c1e4a5bb9d6": {
+        "tool_id": "bandit",
+        "check_id": "yaml-load",
+        "path": "semgrep/semgrep/config_resolver.py",
+        "message": "Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().",
+        "severity": 1,
+        "syntactic_context": "return {config_id: yaml.load(contents, Loader=SafeLineLoader)}"
```

Comment:
we really need to port bandit to get rid of this nonsense

Reply:
why aren't we using safe_load?

Reply:
This is actually a good check! But to get line numbers I had to go around the loader. Very tricky check to not error on what I did...

Reply:
See the code. safe_load is just `load(..., loader=SafeLoader)`. We have a custom loader which tracks line numbers. I think we should switch to `ruamel` but that's a project for another day

Reply:
+100 for ruamel. I use it exclusively for semgrep.live